docs/src/development/ssh/troubleshoot-ssh.md

```diff
@@ -135,9 +142,13 @@ Permission denied (publickey).
 or
 
 ```bash
-GIT_SSH_COMMAND="ssh -v" git clone [REPO-URL]
+GIT_SSH_COMMAND="ssh -v" git clone <REPO_URL>
 ```
 
 You can use this information to make one last check of the private key file.
 
-If you're still stuck, [submit a support ticket](https://console.platform.sh/-/users/:user/tickets) to get help solving your issue.
+## Something still wrong?
+
+{{% troubleshoot %}}
+
+If you're still stuck, [submit a support ticket and provide the full SSH debug information](https://console.platform.sh/-/users/:user/tickets) to get help solving your issue.
```

Comment:
```suggestion
If you're still stuck, [submit a support ticket and provide the full SSH debug information](https://console.platform.sh/-/users/:user/tickets).
```

Just to fit with the style suggestion about not having links in the middle of sentences.